langchain4j-core/src/main/java/dev/langchain4j/store/embedding/filter/MetadataFilterBuilder.java

```diff
@@ -34,6 +35,10 @@ public Filter isEqualTo(String value) {
         return new IsEqualTo(key, value);
     }
 
+    public Filter isEqualTo(UUID value) {
```

Comment:
maybe also `isIn` and `isNotIn` should be supported?

Reply:
good point, done!